python/feathub/processors/spark/tests/test_spark_processor.py

```diff
@@ -152,6 +152,14 @@ def test_bounded_left_table_join_unbounded_right_table(self):
     def test_over_window_transform_count_with_limit(self):
         pass
 
+    # TODO: Add back the following two tests after SparkProcessor support over
+    #  window with filter expression.
+    def test_over_window_transform_filter_expr_with_window_size(self):
+        pass
```

Comment:
If we assume that not many users are using the OverWindow with filters with SparkProcessor, I prefer removing OverWindow with filter until it is supported properly to avoid the different behavior between two processors. What do you think?